### ISSUE TITLE
Run `pscipher -buildkey` to prevent `V1.1` keys from being used

### ISSUE THREAD

**Issue by iversond** [psadmin-io/psadminio-io_weblogic#14]

**Comment by iversond** [comment]
Based on recent security disclosures, the default `V1.1` keys should not be used when encrypting passwords during builds.

**Comment by ericbisme** [comment]
Agreed! We're going to run in to some issues of ordering. For everything else this module does it needs to run after the delivered deployment so that all the pieces exist. By that point the passwords have already been encrypted using the delivered V1.1 keys. We may end up digging in to the type/provider to do this properly.
